### DISCUSSION TITLE
Binding CommandParameter in MAUI

### DISCUSSION THREAD

**Discussion by CaseNewmark** [CommunityToolkit/dotnet#664] [Q&A]
When using the CommunityToolkit.MVVM RelayCommand with MAUI any CommandParameter can only be of type `object`. When doing the same thing on a WinUI project it works.

```
[RelayCommand]
public void DoSomething(SomeType param) { }   // this breaks
```

When doing this it totally messes up any `Binding` in XAML. As an example take the code from the official documentation here:
https://learn.microsoft.com/en-us/dotnet/maui/fundamentals/data-binding/relative-bindings?view=net-maui-7.0

Extent the ´DeleteEmployeeCommand´ to have a CommandParameter of type `Employee`.
Is this intentional? At least it is inconsistent between Winui and Maui.

**Comment by Sergio0694** [comment]
I'm not sure I'm following, what breaks exactly? What error are you getting? Can you share more info to clarify what you mean? Thank you! 🙂
